css/home.css

```diff
@@ -69,29 +70,31 @@ body {
 }
 
 .homepage-hero__article {
-  margin-right: 57px;
-  margin-left: 57px;
+  margin: auto;
+}
+
+.homepage-hero__hdlp-logo{
   position: absolute;
   bottom: 250px;
-  top: 250px;
+  margin-left: -125px;
+  width: 247px;
+  height: 140px;
+  object-fit: contain;
```

Comment:
The logo is not being centered correctly, don't use fixed values like `bottom: 250px`. To center a absolute positioned element vertically and horizontally you could use for example `margin: auto` and `0` value for `top, right, bottom and left` positions.